frontend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:16-alpine AS build
+
+WORKDIR /app
+
+COPY package.json .
+COPY package-lock.json .
+RUN npm install
+RUN npm install -g @ionic/cli
+RUN npm install -g @angular/cli
```

Comment:
tohle je v dev dependencies ("@angular/cli": "~12.1.1"). Podle me, kdyz to pousis pres npm z package.json, nemusis tohle globalne instalovat.
ionic v zavislostech neni, ten je potrebe. I kdyz cistejsi by bylo to mit rovnou v deps a ne globalne.

Reply:
![image](https://user-images.githubusercontent.com/14038818/147921085-af6f7ec7-9618-4e33-8278-de8d15a72be8.png) (coz je teda to co rikas ty, ze tam ionic neni)

tohle je copy pasta z breviary

Reply:
ten angular smazu